examples/basic.rs

```diff
@@ -66,7 +69,9 @@ fn startup(
         array_texture_loader.add(TilemapArrayTexture {
             texture: TilemapTexture::Single(asset_server.load("tiles.png")),
             tile_size,
-            ..Default::default()
+            format: BevyDefault::bevy_default(),
```

Comment:
hmm, these all seem to be using default? Is this not covered by `..Default::default()`?